pkg/metacni/metacni.go

```diff
@@ -207,14 +210,54 @@ func popDevice(devicePool string, allocatedDevices map[string]*[]string)(string,
   return device, nil
 }
 
+func setIpv6Sysctls(args *cniArgs) error {
```

Comment:
can we make this generic?

what I'm thinking is that we anyway need to set a couple of kernel level parameters for multiple reasons in the future, probably. just for IPv6 we already know we need to set some others later on
also setting the parameter is orthogonal to the backend used

how about adding a new list to the DanmEp, something like "kernel_settings"?
and during putDanmEp call we could simply transparently set the full content of this list into the Pod netns

Reply:
code has been refactored heavily...